notes/health-care-evaluation.vbs

```diff
@@ -4925,6 +4985,7 @@ FOR i = 0 to total_clients
 				If HEALTH_CARE_MEMBERS(ref_numb_const, hc_memb) = HH_memb Then
 					HEALTH_CARE_MEMBERS(show_hc_detail_const, hc_memb) = True
 					HEALTH_CARE_MEMBERS(HC_major_prog_const, hc_memb) = "MA"
+					If hc_status_code_past_4_months = False Then HEALTH_CARE_MEMBERS(HC_eval_process_const, hc_memb) = "Recertification"
```

Comment:
I think this is supposed to be "If hc_status_code_past_4_months = True Then ". The logic to get the boolean set correctly, but the 'Recertification' does not come up.

Reply:
Yes, you are 100% correct. Fixed that.